app/medInria/medMainWindow.cpp

```diff
@@ -116,7 +116,7 @@ medMainWindow::medMainWindow ( QWidget *parent ) : QMainWindow ( parent ), d ( n
 
     //  To avoid strange behaviours with the homepageshifter
     this->setMinimumHeight ( 600 );
-    this->setMinimumWidth ( 800 );
+    this->setMinimumWidth ( 950 );
```

Comment:
the minimum width of the software had to be changed since we added a Pipelines column